projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid-navigation.service.ts

```diff
@@ -127,6 +129,23 @@ export class IgxPivotGridNavigationService extends IgxGridNavigationService {
                 layout: null
             }
 
+            if (ctrl) {
+                const dimIndex = this.activeNode.column;
+                const flatRowDImensions = PivotUtil.flatten(this.grid.rowDimensions)
```

Comment:
`visibleRowDimensions` should be used, because if the parent for a previous sibling is collapsed, its child dimensions are not rendered.